plugins/modules/remote_management/redfish/redfish_config.py

```diff
@@ -100,6 +100,12 @@
     type: bool
     default: false
     version_added: 3.7.0
+  hostinterface_config:
+    required: false
+    description:
+      - setting dict of HostInterface on OOB controller
+    type: dict
+    version_added: '3.9.0'
```

Comment:
There will be no 3.9.0 release, the next release (on Tuesday) is 4.0.0.
```suggestion
    version_added: '4.0.0'
```